Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -3360,6 +3372,30 @@ def rtr_get_extracted_file(args_to_get_files: dict, file_name: str):
                            outputs_prefix="CrowdStrike.File"), files]
 
 
+def get_detection_for_incident_command(incident_id: str) -> CommandResults:
+    behavior_res = get_behaviors_by_incident(incident_id)
+    behaviors_id = behavior_res.get('resources')
+
+    if behavior_res.get('meta', {}).get('pagination', {}).get('total', 0) == 0 or not behaviors_id:
```

Comment:
```suggestion
    if behavior_res.get('meta', {}).get('pagination', {}).get('total', 0) == 0 or not behaviors_id:
```
are you sure behavior_res and behavior_res.get('meta', {}).get('pagination', {}) will exist all the time?
maybe better to check if behaviors_id exist first?